compose/config/config.py

```diff
@@ -846,7 +846,7 @@ def resolve_env_var(key, val, environment):
     elif environment and key in environment:
         return key, environment[key]
     else:
-        return key, None
+        return key, ''
```

Comment:
I think this needs to stay `None`. The `None` needs to be handled somewhere else.